resources/lang/en/flights.php

```diff
@@ -5,6 +5,7 @@
     'flighttime'       => 'Flight Time',
     'flighttype'       => 'Flight Type',
     'flighthours'      => 'Flight Hours',
+    'flightcode'       => 'Code',
```

Comment:
just call this `code`. I don't like the above having `flights` on them, kinda redundant which I'll fix later. Otherwise LGTM

Reply:
Not a big deal, we can change it now though. I was in between routecode and flightcode to be honest, used flight code :)